common/src/main/java/com/github/twitch4j/common/util/TypeConvert.java

```diff
@@ -12,9 +15,13 @@
      * ObjectMapper
      */
     @Getter
-    private static final ObjectMapper objectMapper = new ObjectMapper()
-        .registerModule(new ParameterNamesModule())
-        .registerModule(new JavaTimeModule());
+    private static final ObjectMapper objectMapper = JsonMapper.builder()
+        .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS)
+        .enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE)
+        .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+        .propertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE)
```

Comment:
`PropertyNamingStrategy.SNAKE_CASE` is deprecated and replaced by `PropertyNamingStrategies.SNAKE_CASE`